web/ui/react-app/src/pages/targets/Targets.test.tsx

```diff
@@ -10,9 +10,9 @@ describe('Targets', () => {
   };
   const targets = shallow(<Targets {...defaultProps} />);
   it('renders a header', () => {
-    const h1 = targets.find('h1');
-    expect(h1).toHaveLength(1);
-    expect(h1.text()).toEqual('Targets');
+    const h2 = targets.find('h2');
+    expect(h2).toHaveLength(1);
```

Comment:
I would split this test on two - one test to check the count and one the content

Reply:
done